cmd/ping/main.go

```diff
@@ -68,5 +68,17 @@ func main() {
 		})
 		fmt.Fprintln(w, region)
 	})
+
+	// Serve /ping with region response to fix issue#96 on older cli versions.
+	http.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Add("Cache-Control", "no-store")
+		w.Header().Add("Access-Control-Allow-Origin", "*")
+		w.Header().Add("Strict-Transport-Security", "max-age=3600; includeSubdomains; preload")
+		once.Do(func() {
```

Comment:
Let's declare a `pingOnce` and a `apiPingOnce` so these two don't get used together.

Reply:
I suppose so.